compiler/optimizer/OMRSimplifierHandlers.cpp

```diff
@@ -6744,11 +6873,13 @@ TR::Node *daddSimplifier(TR::Node * node, TR::Block * block, TR::Simplifier * s)
    // Pretend to be an int before doing the comparison
    //
    // According to the java spec -0.0 + 0.0 == 0.0
-   // BINARY_IDENTITY_OP(LongInt, DOUBLE_POS_ZERO)
    // In IEEE FP Arithmetic, d + -0.0 is d
    // Pretend to be an int before doing the comparison
    //
-   BINARY_IDENTITY_OP(LongInt, DOUBLE_NEG_ZERO)
+   auto binOpSimplifier = getDoubleBinaryOpSimplifier(s);
```

Comment:
The original code calls macro `BINARY_IDENTITY_OP(LongInt, DOUBLE_NEG_ZERO)` which calls `secondChild->getLongIntType()`. Should the corresponding new API be `getLongBinaryOpSimplifier`?

Reply:
Looks like the comment above the original code explains the reasoning: 

https://github.com/eclipse/omr/blob/f010ed6765cfcf7abc64c34b4097093b9ad4fda3/compiler/optimizer/OMRSimplifierHandlers.cpp#L6743-L6750